paper.md

```diff
@@ -58,202 +58,195 @@ bibliography: paper.bib
+<!-- this will not be part of the final paper -->
+
+We present an open source learning module suitable for a semester long course
+and designed to leverage participatory live-coding techniques to teach both
+statistical and programming skills to primarily upper-year undergraduate biology
+students. Our learning module has three primarily self-contained submodules spanning
+sixteen lessons: 1) Programming in R, basic data wrangling, and visualizations; 2)
+Exploratory data analysis, statistics, and modelling; and 3) Collaborative and reproducible
+science. Our learning module includes eight assignments, distributed throughout
+the term to assess students' learning and understanding of the material. All
+materials were developed and taught using R Markdown, although no prior
+knowledge of the R programming language is assumed or expected. Our material is
+licensed under CC-BY 4.0 while the code is under a MIT License. The open source
+and modular nature of our computational and statistical learning module, as well
+as responding to and emphasizing on the need for greater programming training
+for researchers, makes our material an ideal resource for other instructors and
+we believe it would be an excellent contribution to JOSE.
 
-# Statement of Need
+# Main Body
+
+## Statement of Need
 
 <!--
 Describing why this material is beneficial to the community, why someone else would use it
 -->
 
-In traditional undergraduate biology education, students learn coding skills and
-concepts in biology separately. This course, designed for third and fourth-year
-undergraduate students, fosters R coding skills in the context of learning
+In traditional undergraduate biology education, students learn statistical
+skills and biological concepts separately, often without any teaching on coding
+itself. Designed primarily for upper-year undergraduate students, this learning
+module emphasizes gaining skills in R coding in the context of learning
 statistics and ecology. Notably, the materials cover statistical concepts that
-are broadly useful in biological sciences: linear regression, mixed effects models,
-randomization tests, principal component analyses, ANOVA and MANOVA, model
-selection, and numerically solving differential equations. We delivered these
-materials as a four-month course, but these concepts are presented in stand-alone
-lectures designed by an interdisciplinary teaching team that could easily be
-mixed and matched to suit any desired course outcome. The course is completely
-interactive: all lectures are designed to be delivered in a participatory
-live-coding format so that students learn experientially in real-time. The course
-materials also include assignments matched to lecture materials that sharpen
+are broadly useful in biological sciences, including mixed effects models,
+randomization tests, model selection, and differential equations. While we
+delivered these material and concepts as a four-month semester long course,
```

Comment:
s/material/materials

Reply:
I would consider it singular, as in above. But that does leave 'these' to match the two things a bit weird looking. Not sure if a slight re-arrangement might help.

Reply:
I agree, "material" is more appropriate. I removed "these material" though, so it reads "the material"